lib/app/widgets/category_text_widget.dart

```diff
@@ -0,0 +1,19 @@
+import '../theme/global_imports.dart';
+
+class CategoryTextWidget extends StatelessWidget {
```

Comment:
When you make components, think about sections rather than individual elements.
In the context of a user profile page, it should be like this
```
ProgressSection()
SettingsSection()
```
e.t.c